src/components/structured-text-block/structured-text-block.vue

```diff
@@ -70,16 +70,16 @@
           class: 'structured-text__blue-text',
         })
       }
-      case 'StructuredTextCtaListRecord': {
+      case 'StructuredTextButtonsListRecord': {
         return h('div', {
           key,
-          class: 'structured-text__cta-list',
-        }, record.ctas.map((cta, ctaIndex) => {
+          class: 'structured-text__buttons-list',
+        }, record.buttons.map((button, buttonIndex) => {
           return h(AppButton, {
-            key: ctaIndex,
-            label: cta.title,
-            to: cta.url || cta.link,
-            external: cta.__typename === 'ExternalLinkRecord',
+            key: buttonIndex,
```

Comment:
Could we also query the `id` of the button so we can use that as key? I prefer not to use indexes as keys, as they're not necessarily tied to the actual button.